docs/appendices/release-notes/unreleased.rst

```diff
@@ -209,6 +209,17 @@ Changes
 Fixes
 =====
 
+- Fixed an issue that would lead to incorrect behaviour of the insert from
+  sub query statement for the scenario when the target table contains a
+  :ref:`generated column <sql-ddl-generated-columns>` generated column with
+  the :ref:`not_null_constraint` constraint and a value for the
```

Comment:
```suggestion
  the :ref:`not_null_constraint` and a value for the
```

Reply:
`not_null_constraint` will be rendered as `NOT NULL` only, do you think we have `constraint` here?

Reply:
ah ok sry, ignore it then ;)